src/sass/components/_card.scss

```diff
@@ -161,18 +161,31 @@
     }
 
     &__candidate-read-more-link {
-      left: 0;
       margin-left: 10px;
       top: 0;
-      font-weight: lighter;
+      font-family: $body-font-stack;
+      font-size: 14px;
+      font-weight: 100 !important;
```

Comment:
yes, we are agreed on the goal of wanting to minimize clutter. I know you have said some people seem to not know they can click on things for more context, but IMO 'read/learn more' links should be used much more sparingly and not peppered all over. Part of the clutter is unavoidable when there are this many of them and changing the text styling does help differentiate them, but does not necessarily reduce clutter. We can discuss other ways to better address this later...